tests/utils.py

```diff
@@ -0,0 +1,65 @@
+from typing import Optional, Tuple, List
+from pathlib import Path
+from bip32 import HARDENED_INDEX
+from enum import IntEnum
+
+from ragger.backend import SpeculosBackend
+from ragger.navigator import NavInsID, NavIns
+
+
+def app_path_from_app_name(app_dir, app_name: str, device: str) -> Path:
+    assert app_dir.is_dir(), f"{app_dir} is not a directory"
+    app_path = app_dir / (app_name + "_" + device + ".elf")
+    assert app_path.is_file(), f"{app_path} must exist"
+    return app_path
+
+def prefix_with_len(to_prefix: bytes) -> bytes:
+    return len(to_prefix).to_bytes(1, byteorder="big") + to_prefix
```

Comment:
```suggestion
```
in ragger.utils

Reply:
`prefix_with_len` could use a param for the int size